Systematics/python/SystematicsCustomize.py

```diff
@@ -101,6 +104,9 @@ def modifyTagSequenceForSystematics(process,jetSystematicsInputTags,ZPlusJetMode
     massSearchReplaceAnyInputTag(process.flashggTagSequence,cms.InputTag("flashggDifferentialPhoIdInputsCorrection"),cms.InputTag("flashggDiPhotonSystematics"))
     massSearchReplaceAnyInputTag(process.flashggTagSequence,cms.InputTag("flashggSelectedElectrons"),cms.InputTag("flashggElectronSystematics"))
     massSearchReplaceAnyInputTag(process.flashggTagSequence,cms.InputTag("flashggSelectedMuons"),cms.InputTag("flashggMuonSystematics"))
+    process.flashggWHLeptonicTag.MuonTag = cms.InputTag("flashggMuonSystematicsVH")
```

Comment:
these lines should be modified such that each tag pick up the modified muon collection that corresponds to the ID specified in the MetaConditions